### DISCUSSION TITLE
Duck dns e ip statico

### DISCUSSION THREAD

**Discussion by NuccioT** [indomus/forum#699] [HUB: Home Assistant]
Buongiorno a tutti. Il  mio fornitore di internet mi ha assegnato un ip statico. A questo punto dovrei non dover usare Duckdns. Non so però come fare per ottenere un certificato ssl. Qualcuno può aiutarmi?

**Comment by snakuzzo** [comment]
Non puoi richiedere un certificato su base IP.
Quindi, che sia duckdns o altro, devi per forza avere un tuo dominio.

**Reply by NuccioT** [reply]
Grazie ma la cosa non mi è chiara: la funzione di duckdns dovrebbe essere di "staticizzare l'ip". Se ce l'ho statico pensavo di generare un certificato ssl con putty per ottenere https. Il mio problema è che da maggio l'iphone non si connette più a Hassio da rete domestica e pensavo di poter risolvere il problema con l'ip statico.

**Reply by snakuzzo** [reply]
Per collegarti da remoto (almeno con app nativa) devi per forza avere un certificato rilasciato da una CA, non puoi utilizzarne uno self-signed. La CA (solitamente Let's Encrypt) rilascia i certificati su base dominio, non su base IP.
Che l'IP poi sia statico o dinamico non cambia nulla. Se è statico bene...se non lo è non è un problema e rinnovi l'associazione FQDN/IP tramite script, add-on, automazione o quello che ti pare.

Il problema nasce quando l'ip non è pubblico, ma privato.

Sui tuoi problemi di connessione, andrebbero analizzati nel dettaglio

**Reply by NuccioT** [reply]
Grazie. Per quanto riguarda la connessione iPhone credo il problema sia il certificato che non è più riconosciuto. Da rete locale appare la casa di HA e il cerchio che gira all'infinito

**Reply by snakuzzo** [reply]
Se da locale hai quel comportamento, non credo funzionerà da remoto :)
Devi capire prima qual è il problema in locale...e non si può sapere solo guardando il cerchio che gira.
Ricontrolla le configurazioni di rete
